migrations/1585388899167-add_default_file_storage_provider.js

```diff
@@ -0,0 +1,76 @@
+const mongoose = require('mongoose');
+const CryptoJS = require('crypto-js');
+const { Configuration } = require('@schul-cloud/commons');
+
+const { info, error } = require('../src/logger');
+const { schoolModel: School } = require('../src/services/school/model');
+
+const { connect, close } = require('../src/utils/database');
+
+const StorageProvider = mongoose.model('storageProviderMigration', new mongoose.Schema({
+	type: { type: String, enum: ['S3'], required: true },
+	isShared: { type: Boolean },
+	accessKeyId: { type: String, required: true },
+	secretAccessKey: { type: String, required: true },
+	endpointUrl: { type: String, required: true },
+	region: { type: String },
+	maxBuckets: { type: Number, required: true },
+	schools: [{ type: mongoose.Schema.Types.ObjectId, ref: 'school' }],
+}, {
+	timestamps: true,
+}), 'storageproviders');
+
+module.exports = {
+	up: async function up() {
+		await connect();
+		const S3_KEY = Configuration.get('S3_KEY');
+
+		if (!process.env.AWS_SECRET_ACCESS_KEY) {
```

Comment:
```suggestion
		if (!Config.has('AWS_SECRET_ACCESS_KEY')) {
```
nowhere use process.env :)

Reply:
Bevor Cedric's PR nicht durch ist, ist das hier best practice gewesen und daher bisher okay. Den Config-Key gibt es noch nicht.